clients/admin-ui/package.json

```diff
@@ -16,6 +16,7 @@
     "analyze:server": "cross-env BUNDLE_ANALYZE=server next build",
     "analyze:browser": "cross-env BUNDLE_ANALYZE=browser next build",
     "export": "next build && next export",
+    "copy-export": "mkdir -p ../../src/fides/ui-build/static/admin/ && rsync -a --delete out/ ../../src/fides/ui-build/static/admin/",
```

Comment:
this doesn't work on windows but I'm assuming it does on macs...I'll have someone else check it though